services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -444,6 +444,26 @@ module.exports = {
       return isArray(sectionQuery) ? sectionQuery : [];
     },
 
+    /**
+     * Return the oldest start date for the current section
+     */
+    sectionStartDate: async (content, { input }, { basedb }) => {
+      const { sectionQuery } = content;
+      const { sectionId } = input;
+      const descendantIds = await getDescendantIds(sectionId, basedb);
+      const sectionSchedules = sectionQuery.filter(section => section.sectionId === sectionId);
+      const descendantSectionSchedules = sectionQuery
+        .filter(section => descendantIds.includes(section.sectionId))
+        .map(section => section.start);
+      const sectionDates = [
+        ...new Set([
+          ...sectionSchedules.map(section => section.start),
+          ...descendantSectionSchedules,
+        ]),
+      ];
+      return sectionDates[0];
```

Comment:
We don't care about uniqueness, you should just be able to do something like this
```js
  return schedules.length ? schedules[0].start : null;
```